build.gradle

```diff
@@ -22,6 +21,11 @@ allprojects {
     }
 }
 
+subprojects {
+    apply plugin: 'idea'
+    apply from: "${rootDir}/gradle/dependencies.gradle"
```

Comment:
Do we need to invoke `apply from: "${rootDir}/gradle/dependencies.gradle"` twice in this file?